src/Form/Field/FormField.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import classNames from "classnames";
+import styled from "styled-components";
+
+const HelperMessage = styled.p`
+  margin: 0;
+
+  color: gray;
+`;
+const ErrorMessage = styled.p`
+  margin: 0;
+
+  color: red;
+`;
+
+interface FormFieldProps {
+  labelledBy: string;
+  label?: string;
+  className?: string;
+  children: any;
+  helperMessage?: string;
+  errorMessage?: string;
+}
+
+function FormField(props: FormFieldProps) {
+  const {labelledBy, label, className, children, errorMessage, helperMessage} = props;
+  const hasLabel = Boolean(label);
+  const hasErrorMessage = Boolean(errorMessage);
+  const hasHelperMessage = Boolean(helperMessage);
+  const formFieldClassName = classNames(
+    "form-field",
+    {
+      "has-error": hasErrorMessage
+    },
+    className
+  );
+
+  return (
+    <div className={formFieldClassName}>
+      {hasLabel && (
+        <label id={labelledBy} className={"form-field-label"}>
+          {label}
+        </label>
+      )}
+
+      {children}
+
+      {hasErrorMessage && <ErrorMessage>{errorMessage}</ErrorMessage>}
```

Comment:
We have `warningMessage` and `errorMessage` at Wolarf. `helperMessage === warningMessage`?

Reply:
There is a small difference between warning and helper. 

The helper message gives you a small tip about the field for example, "Best usernames are about 5 characters." is a helper message because it gives you a small tip but doesn't restrict you to use username longer than 5 characters. 

In this case, warning/error message would be "Please use a username shorter than 5 characters" and probably you're gonna restrict the user.